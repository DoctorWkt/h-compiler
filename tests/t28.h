void main(int argc, char *argv)
{
  int i;
  for (i=-3; i != 4; i++) {
    printn(i);
    printn(i && 1);
    putchar('\n');
  }
}
