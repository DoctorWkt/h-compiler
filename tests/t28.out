777775
000001

777776
000001

777777
000001

000000
000000

000001
000001

000002
000001

000003
000001

